packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4528,6 +4563,20 @@ class SemanticsConfiguration {
     _setFlag(SemanticsFlag.isHeader, value);
   }
 
+  /// Indicates the heading level in the document structure.
+  ///
+  /// This is only used for web semantics, and is ignored on other platforms.
+  int? get headingLevel => _headingLevel;
+  int? _headingLevel;
+
+  set headingLevel(int? value) {
```

Comment:
should probably guard around the input range?

Reply:
Good point, done.